app/views/users/sessions/new.html.erb

```diff
@@ -2,29 +2,25 @@
   <h2>Log in</h2>
 
   <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-    <div class="field grid-field">
-      <%= f.label :username %>
-      <%= f.text_field :username, autofocus: true, autocomplete: "username" %>
+    <div class="icon-wrapper user-field">
+      <%= f.text_field :username, class: 'text-field', placeholder: 'Username or email', maxlength: '40' %>
     </div>
-
-    <div class="field grid-field">
-      <%= f.label :password %>
-      <%= f.password_field :password, autocomplete: "current-password" %>
+    <div class="icon-wrapper password-field">
+      <%= f.password_field :password, class: 'text-field', placeholder: 'Password', maxlength: '40' %>
     </div>
 
-    <div class="flex-column-centred">
-      <% if devise_mapping.rememberable? %>
-        <div class="field">
-          <%= f.check_box :remember_me %>
-          <%= f.label :remember_me %>
-        </div>
-      <% end %>
-
-      <div class="actions submit-button">
-        <%= f.submit "Log in" %>
+    <% if devise_mapping.rememberable? %>
+      <div class="field">
+        <%= f.check_box :remember_me %>
+        <%= f.label :remember_me %>
       </div>
-    </div>
+    <% end %>
+
+    <div id='user_submit' class="general-button glaring">Sign in/up</div>
```

Comment:
Having sign in/up button together means there's no easy way to get directly to the sign up form. The user has to enter a username and password to get past the form validations before they can click the button. Could separate the buttons or populate the sign up form with the data the user entered